miniapps/electromagnetics/maxwell_solver.hpp

```diff
@@ -0,0 +1,184 @@
+// Copyright (c) 2010, Lawrence Livermore National Security, LLC. Produced at
+// the Lawrence Livermore National Laboratory. LLNL-CODE-443211. All Rights
+// reserved. See file COPYRIGHT for details.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability see http://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the GNU Lesser General Public License (as published by the Free
+// Software Foundation) version 2.1 dated February 1999.
+
+#ifndef MFEM_MAXWELL_SOLVER
+#define MFEM_MAXWELL_SOLVER
+
+#include "../../config/config.hpp"
+
+#ifdef MFEM_USE_MPI
+
+#include "mfem.hpp"
+#include "../common/pfem_extras.hpp"
+
+using namespace std;
+using namespace mfem;
+
+namespace mfem
+{
+
+using miniapps::ND_ParFESpace;
+using miniapps::RT_ParFESpace;
+using miniapps::ParDiscreteCurlOperator;
+
+namespace electromagnetics
+{
+
+// Physical Constants
+
+// Permittivity of Free Space (units F/m)
+static double epsilon0_ = 8.8541878176e-12;
```

Comment:
It would make sense for these to be const.

Reply:
An excellent point!

It would also make sense to move these into a shared header file rather than duplicating them in the various minapps.  I just made both of these changes.